.gitignore

```diff
@@ -1,3 +1,16 @@
 .env
 node_modules/
 dist/
+
+## used for some private servers, dist/**.js concated into build
+# make shure requrie.js is not concated and main.js is concated at last
+build/
+
+## favorite idea
+.idea
```

Comment:
+1

You can also use .git/info/exclude for per-project environment ignores. Otherwise the project's .gitignore would contain lines from every known editor.